test/rpmtest/manifest.go

```diff
@@ -44,15 +45,28 @@ func PackagesFromRPMManifest(t *testing.T, r io.Reader) []*claircore.Package {
 			RepositoryHint: "key:" + rpm.GPG,
 			Module:         rpm.Module,
 		}
-		if s, ok := src[rpm.Source]; ok {
+
+		// Newer images produced from Konflux shove all the source information
+		// into the SourceName and omit the SourceNEVRA. Try both.
+		var source string
+		switch {
+		case rpm.SourceNEVRA != "":
+			source = rpm.SourceNEVRA
+		case rpm.SourceName != "":
+			source = rpm.SourceName
```

Comment:
is it possible for both to be populated?

Reply:
Yes, older manifests will have `SourceNEVRA` populated, newer ones won't but they have the same information in `SourceName`.